tools/git_lfs.rst

```diff
@@ -52,8 +54,6 @@ First, add these lines into your :file:`~/.gitconfig` file:
 
 .. literalinclude:: snippets/git_lfs_gitconfig.txt
```

Comment:
@jonathansick I just updated the server with the new API changes. So now if you remove the batch = False it should not warn.